src/LatteTemplateResolver/NetteApplicationUIPresenterStandalone.php

```diff
@@ -24,7 +24,7 @@ protected function getTemplatePathPatterns(ReflectionClass $reflectionClass, str
 
         return [
              $dir . '/templates/' . $presenterName . '/([a-zA-Z0-9_]*?).latte',
-             $dir . '/templates/' . $presenterName . '\.([a-zA-Z0-9_]?).latte',
+             $dir . '/templates/' . $presenterName . '\.([a-zA-Z0-9_]*?).latte',
```

Comment:
Shouldn’t be `+` there? And also few lines above too?

Reply:
Yeah that would be better